packages/design-system/src/themes/default/colors.css

```diff
@@ -35,6 +35,20 @@
   --ads-color-black-0 : #FFFFFF;
 
 
+  /* gray */
```

Comment:
@tanvibhakta / @albinAppsmith : Shouldn't we will follow our new naming convention in greyscale? As we are changing variables across the ads & same variable will be used in main repo

Reply:
This is just for the colors. Variable naming convention is for global and component level variables.